.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+/lua/nvim-treesitter/textobjects/ @theHamsta
+/lua/nvim-treesitter/incremental_selection.lua @theHamsta
```

Comment:
doesn't @kyazdani42 want to take care of this?

Reply:
We'll discuss that later, that's just a draft